modules/google-storage-bucket/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "this" {
+  description = "A bucket in Google cloud storage service (GCS)."
```

Comment:
very small thing, but since "Cloud Storage" is the full name of the service (afaik) and our phrasing has generally been <Service> <resource> we could just say `Google Cloud Storage bucket`, no?